lib/modules/versioning/distro.ts

```diff
@@ -35,12 +39,30 @@ export class DistroInfo {
       const schedule = this._distroInfo[version];
       this._codenameToVersion.set(schedule.series, { version, ...schedule });
     }
+
+    const arr = Object.keys(this._distroInfo).sort(
+      (a, b) => parseFloat(a) - parseInt(b)
+    );
+
+    for (const v of arr) {
+      const obj = { version: v, ...this._distroInfo[v.toString()] };
+      if (!obj.eol) {
+        // istanbul ignore next
+        continue;
+      }
+      this._sortedInfo.push(obj);
+    }
   }
 
   public isCodename(input: string): boolean {
     return this._codenameToVersion.has(input);
   }
 
+  public exists(input: string): boolean {
```

Comment:
can you please add tsdoc comments, so it's more clear what those functions are doing.

would be useful for future development by other community members